src/Components/Components/src/Binding/IFormValueSupplier.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Components.Binding;
+
+/// <summary>
+/// Binds form data valuesto a model.
+/// </summary>
+public interface IFormValueSupplier
+{
+    /// <summary>
+    /// Determines whether the specified value type can be bound.
+    /// </summary>
+    /// <param name="formName">The form name to bind data from.</param>
+    /// <param name="valueType">The <see cref="Type"/> for the value to bind.</param>
+    /// <returns></returns>
+    bool CanBind(string formName, Type valueType);
```

Comment:
```suggestion
    /// <summary>
    /// Determines whether the specified value type can be bound.
    /// </summary>
    /// <param name="formName">The form name to bind data from.</param>
    /// <param name="valueType">The <see cref="Type"/> for the value to bind.</param>
    /// <returns><c>true</c> if the value type can be bound; otherwise, <c>false</c>.</returns>
    bool CanBind(string formName, Type valueType);
```